src/database.ts

```diff
@@ -30,22 +31,24 @@ import * as is from 'is';
 import * as r from 'request';
 import * as streamEvents from 'stream-events';
 import * as through from 'through2';
-
-import {BatchTransaction} from './batch-transaction';
-import {codec} from './codec';
+import {Operation as GaxOperation} from 'google-gax/build/src/longRunningCalls/longrunning';
+import {BatchTransaction, TransactionIdentifier} from './batch-transaction';
+import {google as database_admin_client} from '../proto/spanner_database_admin';
 import {Instance} from './instance';
+import {PartialResultStream} from './partial-result-stream';
```

Comment:
```suggestion
import {PartialResultStream, Row} from './partial-result-stream';
```
we can use `Row` in `RunCallback`

Reply:
Done!